test/stateless/deployment.test.ts

```diff
@@ -182,7 +182,20 @@ function applyNagSuppression(stackId: string, stack: Stack) {
         ]
       );
       break;
-
+    case 'Cttsov2Icav2PipelineManagerStack':
+      NagSuppressions.addResourceSuppressions(
+        stack,
+        [
+          {
+            id: 'AwsSolutions-IAM5',
+            reason:
+              'The provider function needs to be able to invoke the configured function. It uses' +
+              "`lambda.Function.grantInvoke` to achieve this which contains a '*' and is not changeable.",
+          },
+        ],
+        true
+      );
+      break;
```

Comment:
Perhaps, this comes rebasing from main... I reckon

Reply:
Haha, I added it before as there is an issue when I test the stateless stacks, but it seems fixed in recent commits.
Already delete it.